cypress/support/fragments/circulation/newNoticePolicyTemplate.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable cypress/no-unnecessary-waiting */
 import getRandomPostfix from '../../utils/stringTools';
-import { Button, TextField, TextArea, KeyValue, Checkbox, Link, Heading, Select, Pane, Modal, Section } from '../../../../interactors';
+import { Button, TextField, TextArea, KeyValue, Checkbox, Link, Heading, Select, Pane, Modal, Section, NavListItem } from '../../../../interactors';
```

Comment:
'NavListItem' is defined but never used. Allowed unused vars must match /React/u

Reply:
removed